src/lib.rs

```diff
@@ -1,12 +1,13 @@
 #![no_std]
 #![feature(const_fn)]
+#![feature(panic_implementation)]
 #![allow(non_upper_case_globals)]
 #![feature(lang_items)]
```

Comment:
I don't think lang_items is needed anymore.

Reply:
I'll try without it